graftm/archive.py

```diff
@@ -0,0 +1,114 @@
+import tarfile
+import logging
+import itertools
+import os
+import tempdir
+import shutil
+
+from graftm_package import GraftMPackage
+
+class ArchiveDefaultOptions:
+    force=False
+
+class Archive:
+    def create(self, input_package_path, output_package_path, **kwargs):
+        """Create an archived GraftM package
+
+        Parameters
+        ----------
+        input_package: GraftMPackage
+            Package to be archived
+        output_pacakge_path: str
+            output package path
+        kwargs:
+            force: bool
+                overwrite an existing directory
+
+        Returns
+        -------
+        nothing"""
+        force = kwargs.pop('force',ArchiveDefaultOptions.force)
```

Comment:
I think you mean instance variable not global variable? Either way, I'd prefer not to make things have more scope than necessary.